plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -165,6 +190,21 @@ export function createPublishGithubAction(options: {
         });
       }
 
+      if (collaborators) {
+        for (const {
+          access: permission,
+          username: team_slug,
+        } of collaborators) {
+          await client.teams.addOrUpdateRepoPermissionsInOrg({
```

Comment:
Thinking it would be best to log errors here, but continue the process instead of throwing

Reply:
Added try/catch + logging + test